src/anshuKJun20/Bank.java

```diff
@@ -0,0 +1,78 @@
+/*Assignment 8 : 16th Jun 2020
+
+Create a Banking System which has following functionality.
+
+Create two object to perform below scenario.
+1) Method to Debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output:
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variable will be required.
+Output:
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time */
+
+
+package anshuKJun20;
+
+public class Bank {
+	int totalCA;
+	int totalDA;
+	int currentbalance;
+   static int credittimes=0;
+   static int debitttimes=0;
+   static int printbalnc=0;
+   
+   //Here debit 1 and 2 means user debited different amount in 2 times.
+   void debitAmount(int debit1, int debit2, int debit3,int debit4){
+		int debitamount = debit1 +debit2+debit3+debit4;
+		totalDA=debitamount;
+	}
+   //Here debit 1 and 2 means user credited different amount in 5 times.
+	void creditAmount(int credit1,int credit2, int credit3, int credit4, int credit5){
```

Comment:
First understand the requirement properly, because your entire code is wrong. Please refer to Aastha's code.
https://github.com/KrishnaKTechnocredits/JAVATechnoJun20/pull/110